src/main/java/com/dabsquared/gitlabjenkins/GitLabMergeRequest.java

```diff
@@ -60,6 +63,18 @@ public String toString() {
         return ToStringBuilder.reflectionToString(this, ToStringStyle.MULTI_LINE_STYLE);
     }
 
+    public GitlabCommitStatus createCommitStatus(GitlabAPI api, String status, String targetUrl) {
+        try {
+            if(objectAttributes.lastCommit!=null) {
+                return api.createCommitStatus(sourceProject, objectAttributes.getLastCommit().getId(), status, objectAttributes.getLastCommit().getId(), "Jenkins", targetUrl, null);
```

Comment:
@EmteZogaf can you submit a PR for this so it doesn't get lost as just a comment on the source? Thanks!